tests/kafkatest/tests/streams/streams_standby_replica_test.py

```diff
@@ -100,6 +100,10 @@ def test_standby_tasks_rebalance(self):
         self.wait_for_verification(processor_2, "ACTIVE_TASKS:6 STANDBY_TASKS:0", processor_2.STDOUT_FILE)
 
         processor_3.start()
+
+        self.wait_for_verification(processor_3, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_3.STDOUT_FILE)
+        self.wait_for_verification(processor_2, "ACTIVE_TASKS:3 STANDBY_TASKS:3", processor_2.STDOUT_FILE, num_lines=2)
```

Comment:
I'm not 100% sure I get what's going on here, but it looks like this is the 4th time processor 2 should have this line? Should it be num_lines=4  (and num_lines=3 on L96?)

Likewise, should processor 3 be num_lines=3?

Reply:
@vvcephei The log files are deleted when a processor is stopped and started (note, this does not apply if one does a `processor.restart()` but only on explicit `processor.stop()` plus `processor.start()`). Thus, first occurrence in the log is L96 and this is second one).